### ISSUE TITLE
Disappearing messages after switching conversations.

### ISSUE THREAD

**Issue by PicoGeyer** [wireapp/wire-desktop#703]
Wire Version: 2.13.2741
Wire for Web Version: 2017.06.07.1805
OS: <Arch linux>
Do you have an antivirus software installed: No

What steps will reproduce the problem?
1. Post a message to Otto "Test message 1" (any message :) ) 
2. Switch to another conversation.
3. Switch back to the conversation with Otto. The last message sent should have vanished. 

What is the expected result?
Expecting to see the message that I just sent.

What happens instead?
My previously sent message has now vanished which makes the conversation hard to follow.

![wire_bug3](https://user-images.githubusercontent.com/1290303/27328131-513c8106-55b1-11e7-9b6b-01a7e6bd4366.png)
After switching back to the conversation:
![wire_bug4](https://user-images.githubusercontent.com/1290303/27328144-5823b20a-55b1-11e7-991b-bbc0115c6dba.png)

**Comment by gregor** [comment]
We have been unable to reproduce this internally. As we have made many changes in this area in the past weeks I am going ahead and closing this issue at this point in time.

**Comment by PicoGeyer** [comment]
Unfortunately the issue is definitely still present for me.
I'm usually on Arch linux, but I tested now on Ubuntu 16.04 using the official package from( https://medium.com/@wireapp/a-step-forward-for-wire-for-linux-52f0538cac15 )  and I get the same issue.

**Comment by PicoGeyer** [comment]
After some more testing, it seems the issue is only specific to my one machine. Very strange because it happens even inside an VM on that machine. Let me know if I can provide further info or if there are any other suggestions. I understand it's not worth developer time if I'm the only one with the issue.
